dbunit/src/main/java/org/jboss/arquillian/persistence/dbunit/dataset/scriptable/ScriptableTable.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2010, Red Hat Middleware LLC, and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.arquillian.persistence.dbunit.dataset.scriptable;
+
+import org.dbunit.dataset.DataSetException;
+import org.dbunit.dataset.ITable;
+import org.dbunit.dataset.ITableMetaData;
+
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+import java.util.regex.Pattern;
+
+/**
+ * @author <a href="mailto:rmpestano@gmail.com">Rafael Pestano</a>
+ *         <p/>
+ *         Adds support for script language (JSR 223) in table values.
+ */
+public class ScriptableTable implements ITable {
+
+    //any non digit char followed by ':' followed by 1 or more chars e.g: js: new Date().toString()
+    private final Pattern scriptEnginePattern = Pattern.compile(".*\\D.*:.+");
+
+    private static Logger log = Logger.getLogger(ScriptableTable.class.getName());
+
+    private Map<String, ScriptEngine> engines;
+
+    private ScriptEngineManager manager;
+
+    private ITable delegate;
+
+    private ScriptableTable() {
+        throw new UnsupportedOperationException("This constructor should not be called");
```

Comment:
That's from a 'code style' from other project which doesn't make sense here. Will fix it.